fineract-provider/src/main/java/org/apache/fineract/useradministration/domain/AppUser.java

```diff
@@ -230,7 +230,7 @@ public void updateRoles(final Set<Role> allRoles) {
         final String passwordEncodedParamName = "passwordEncoded";
         if (command.hasParameter(passwordParamName)) {
             if (command.isChangeInPasswordParameterNamed(passwordParamName, this.password, platformPasswordEncoder, getId())) {
-                final String passwordEncodedValue = command.passwordValueOfParameterNamed(passwordParamName, platformPasswordEncoder,
```

Comment:
why remove this `final` ? It doesn't seem to do any harm, and prevents the variable from being changed again by mistake.